man/zeek.8

```diff
@@ -148,6 +150,28 @@ Output file for script execution statistics
 .TP
 .B ZEEK_DISABLE_ZEEKYGEN
 Disable Zeekygen (Broxygen) documentation support
+.SH OUTPUT FORMAT
+Output is written in multiple files depending on configuration. Default
+location is the current directory. Packets can be written to a tcpdump file.
```

Comment:
I am a tiny bit hesitant to actually mention the tcpdump file here. That is code that is not executed rarely - and I would not be surprised if it was broken in random ways.

Reply:
ohhh, I have removed last part.